libs/libvtrutil/src/vtr_vec_id_set.h

```diff
@@ -2,6 +2,7 @@
 #define VTR_SET_H
 
 #include <vector>
+#include <algorithm>
```

Comment:
Why did you include the algorithm library here? If it is for an implementation (.cpp) file it should probably be there instead.